wiki/Accuracy/id.md

```diff
@@ -0,0 +1,76 @@
+# Akurasi
+
+Akurasi adalah suatu pengukur konsistensi pemain. Terdapat tiga aipe akurasi yang dipunyai pemain. Salah satunya yaitu akurasi beatmap yang bergantung pada skor hit yang diperoleh. Lainnya adalah akurasi rata-rata pemain yang ditimbang agar skor yang lebih baik dapat lebih menonjol. Dan terakhir, akurasi [pp](/wiki/pp) pemain yang bergantung pada akurasi skor yang terkirim.

```

Comment:
(this suppose to be tipe right?) aipe -> jenis
dipunyai -> dimiliki

Reply:
yes